OrbitGl/App.cpp

```diff
@@ -317,10 +317,6 @@ void OrbitApp::PostInit() {
 
   GCurrentTimeGraph->SetStringManager(string_manager_);
 
-  if (!absl::GetFlag(FLAGS_enable_tracepoint_feature)) {
```

Comment:
Since this is effectively enabling this feature for everyone, this change should be done in a different PR.